include/SDL3/SDL_video.h

```diff
@@ -318,7 +318,8 @@ typedef enum SDL_ProgressState
     SDL_PROGRESS_STATE_INDETERMINATE,   /**< The progress bar is shown in a indeterminate state */
     SDL_PROGRESS_STATE_NORMAL,          /**< The progress bar is shown in a normal state */
     SDL_PROGRESS_STATE_PAUSED,          /**< The progress bar is shown in a paused state */
-    SDL_PROGRESS_STATE_ERROR            /**< The progress bar is shown in an error state */
+    SDL_PROGRESS_STATE_ERROR,           /**< The progress bar is shown in an error state */
```

Comment:
```suggestion
    SDL_PROGRESS_STATE_ERROR,           /**< The progress bar is shown in a state indicating the application had an error */
```
Is that correct?

Reply:
Yes, it changes the progress bar to red to signal an error in the user application, not in the taskbar API.